exchanges/huobi/huobi_websocket.go

```diff
@@ -12,6 +12,7 @@ import (
 	"text/template"
```

Comment:
PSA for any other reviews with old fields like me. If you have a subscription called `account` its changed to `myAccount` and this will fail. Just delete all subs and they'll be re-added